test/hotspot/jtreg/compiler/vectorapi/VectorFPtoIntCastTest.java

```diff
@@ -84,7 +84,7 @@ public VectorFPtoIntCastTest() {
     }
 
     @Test
-    @IR(counts = {"F2X", ">= 1"})
+    @IR(counts = {"VectorCastF2X", "> 0"})
```

Comment:
You can directly use `IRNode.VECTOR_CAST_F2X` (and `IRNode.VECTOR_CAST_D2X` below) which makes it clearer that this is an actual IR node and not a custom string.

Reply:
@chhagedorn I made the change accordingly.